src/db/models/JobModel.ts

```diff
@@ -177,4 +191,20 @@ export class JobModel extends Model<JobModel> {
     onDelete: "CASCADE",
   })
   applications: ApplicationModel[];
+
+  @AfterCreate
+  static async sendEmailHook(instance: JobModel) {
+    console.log("New entry created");
+    const mailerService = new MailerService();
+
+    const dto: SendEmailDto = {
+      from: { name: "TPC Portal", address: "aryangkulkarni@gmail.com" },
+      recepients: [{ address: "me210003016@iiti.ac.in" }],
+      subject: "Test email",
+      html: "<p>Hi Aryan, this is a test email</p>",
```

Comment:
Add some meaningful message as per the context. This will be getting merged directly in the main.